diff --git a/node_modules/@stencil/core/compiler/stencil.js b/node_modules/@stencil/core/compiler/stencil.js
index aa7fe89..2e0da39 100644
--- a/node_modules/@stencil/core/compiler/stencil.js
+++ b/node_modules/@stencil/core/compiler/stencil.js
@@ -226142,7 +226142,7 @@ const canSkipGlobalStyles = async (config, compilerCtx, buildCtx) => {
   if (buildCtx.isRebuild && !buildCtx.hasStyleChanges) {
     return true;
   }
-  if (buildCtx.filesChanged.includes(config.globalStyle)) {
+  if (buildCtx.filesChanged.includes(normalizePath$2(config.globalStyle))) {
     // changed file IS the global entry style
     return false;
   }
